vasern/src/core/vasern/Document.js

```diff
@@ -57,7 +57,7 @@ export default class Document {
     // initate _data
     this._data = [];
 
-    this.bindEvents.bind(this)();
+    this.bindEvents.bind(this)(); // Question: Is there a reason for the double ()?
```

Comment:
This line support to call the `bindEvents`. The first "()" is for binding the method to the current `Document` class, the second one is actually calling the method.

Reply:
Oh yeah, good point. Question resolved :+1: